interop-testing/src/main/java/io/grpc/testing/integration/TestServiceClient.java

```diff
@@ -280,7 +283,11 @@ protected ManagedChannel createChannel() {
             .sslContext(sslContext)
             .build();
       } else {
-        OkHttpChannelBuilder builder = OkHttpChannelBuilder.forAddress(serverHost, serverPort);
+        OkHttpChannelBuilder builder = OkHttpChannelBuilder.forAddress(serverHost, serverPort)
+            .connectionSpec(new ConnectionSpec.Builder(OkHttpChannelBuilder.DEFAULT_CONNECTION_SPEC)
+                .cipherSuites(TestUtils.preferredTestCiphers().toArray(new String[0]))
```

Comment:
No. We want to use the defaults. This is supposed to represent what a user would do.